### DISCUSSION TITLE
Create nested bindings

### DISCUSSION THREAD

**Discussion by txtyash** [gdh1995/vimium-c#766] [Q&A] [answered]
I have mapped `s` to open links but I can't yank links as I can do with `f`.

Basically I want to do `ys<hint>` which doesn't yank the link but instead opens it. I want it to work like `yf<hint>`

**Comment by gdh1995** [comment]
Sorry but I can't understand the "yank" word in your question. What're your key mappings?

Vimium C has supported the feature of mapping a sequence of keys to a command, such as:
```
map ys LinkHints.activate
```

**Reply by gdh1995** [reply]
And, what did you mean by `I can do with f`, `ys` or `yf` ?

**Reply by txtyash** [reply]
Oh, I meant copying(yanking) links to the clipboard. My current mappings for link hints are:
```
map s LinkHints.activate
map S LinkHints.activateOpenInNewTab
```
If I have to copy a link to the clipboard I cannot use `ys<link_hint>` while I can copy the link with `yf<link_hint>`.
So, I want to be able to copy the links with `ys<link_hint>`

My entire mappings list in case:
```
map s LinkHints.activate
map S LinkHints.activateOpenInNewTab
map , moveTabLeft
map . moveTabRight
map r reload hard
map ( previousTab
map ) nextTab
map { previousTab count=2
map } nextTab count=2
map o Vomnibar.activateInNewTab
map O Vomnibar.activate
map ^ firstTab
map $ lastTab
map e visitPreviousTab
map % togglePinTab
map f Vomnibar.activateTabSelection currentWindow
map + zoomIn
unmap -
map - zoomOut
map = zoomReset
map ! closeDownloadBar

unmap <<
unmap >>
```

**Comment by gdh1995** [comment, marked as answer]
Why did you say "I cannot use `ys...`" ?

What will it do if you add `map ys LinkHints.activateModeToCopyLinkUrl` ?

**Reply by txtyash** [reply]
Yeah, my bad I should've seen the available commands thoroughly.
